test/integration/other/http-headers.js

```diff
@@ -0,0 +1,9 @@
+const should = require('should');
+const { testService } = require('../setup');
+
+describe('http-headers', () => {
+  it('should not include x-powered-by header', testService(async (service) => {
+    const { headers } = await service.get('/v1/users/current');
+    should(headers['x-powered-by']).be.undefined();
+  }));
+});
```

Comment:
Looks familiar :grin:

Reply:
Haha, yeah, I cherry picked. [The commit ](https://github.com/getodk/central-backend/pull/1111/commits/ad0e2fc0b0426635d113244e9e34a6e3d3757e18) says it's authored by you so not sure why it's showing my face!